app/email_branding/email_branding_schema.py

```diff
@@ -7,6 +7,7 @@
     "properties": {
         "colour": {"type": ["string", "null"]},
         "name": {"type": "string"},
+        "alt_text": {"type": "string"},
```

Comment:
It feels like this could match the other nullable database fields and be `{"type": ["string", "null"]}`, but I think this is fine if we're never planning on passing None through from admin

Reply:
helpfully, this now _does_ need to be `null`. i'll update this PR

Reply:
commits squashed.